torch_brain/nn/rotary_attention.py

```diff
@@ -467,6 +486,7 @@ def rotary_attn_xformers_func(
         value,
         attn_bias=attn_bias,
         p=dropout_p,
+        op=(xops.fmha.cutlass.FwOp, xops.fmha.cutlass.BwOp),
     )
```

Comment:
This definitely feels like bad practice. We should find another approach so that we don't set the op like this. I think slightly older version of xformers did not have this problem.

If we have to resolve to fixing the op, then we should atleast set it to flash when bf16 is detected.